misc/TinyFrame/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "TinyFrame",
+    "description": "TinyFrame: Event-driven button lib for embedded",
```

Comment:
"TinyFrame: Event-driven button lib for embedded" ?

But the previous description is ,

"TinyFrame: Serial communication protocol."